.github/workflows/deploy-site.yml

```diff
@@ -31,8 +31,8 @@ jobs:
       - run: yarn run test
       - run: yarn run build
         env:
-          PUBLIC_CAPTCHA_KEY: 6LdU5kckAAAAANktvvAKJ0auYUBRP0su94G7WXwe
-          PUBLIC_FAUCET_URL: https://ink-docs-rococo-faucet.parity-testnet.parity.io/drip/web
+          PUBLIC_CAPTCHA_KEY: 6LesXHomAAAAAGVVTCgc467t8hvBbmK7IlYZCc8O
+          PUBLIC_FAUCET_URL: https://faucet.rococo.frequency.xyz/drip/web
```

Comment:
Wouldn't this need to be something different if we are targeting the cname `faucet.rococo.frequency.xyz` to github pages? Perhaps `faucet-api.rococo.frequency.xyz`?

Reply:
it depends how we are going to do the routing in our load balancer but yeah, maybe changing it would make deployment easier

Reply:
Applied the change